CONTRIBUTING.md

```diff
@@ -2,45 +2,138 @@
 
 Hi there! We're excited to have you as a contributor.
 
-If you have questions about this document or anything not covered here?
-See the [Community section](https://ansible.readthedocs.io/projects/runner/en/latest/community/) of the docs for information about getting in touch.
+If you have questions about this document or anything not covered here,
+see the [Community section](https://ansible.readthedocs.io/projects/runner/en/latest/community/) of the docs for
+information about getting in touch.
+
+## Prerequisites
+
+Before you start contributing, ensure you have:
+
+- Python 3.10 or later installed
+- Git configured with your name and email
+- A GitHub account for submitting pull requests
 
 ## Things to know prior to submitting code
 
 - All code and doc submissions are done through pull requests against the `devel` branch.
 - Take care to make sure no merge commits are in the submission, and use `git rebase` vs `git merge` for this reason.
-- We ask all of our community members and contributors to adhere to the [Ansible code of conduct]. If you have questions, or need assistance, please reach out to our community team at [codeofconduct@ansible.com].
+- We ask all of our community members and contributors to adhere to the [Ansible code of conduct]. If you have questions,
+  or need assistance, please reach out to our community team at [codeofconduct@ansible.com].
 
 ## Setting up your development environment
 
-In this example we are using [virtualenvwrapper](https://virtualenvwrapper.readthedocs.io/en/latest/), but any virtual environment will do.
+The recommended way to set up a development environment is using `tox --devenv` which creates a virtual environment
+with all necessary dependencies. You will need to either install it with your system package manager, or use `pip`:
+
+```console
+$ pip install tox
+```
+
+Once installed, create your own development environment and activate it with:
+
+```console
+$ tox --devenv venv
+$ source venv/bin/activate
+```
+
+This creates a virtual environment in the `venv` directory with ansible-runner installed in development mode along
+with all dependencies.
```

Comment:
Note that you can select the environment just like with other subcommands. This would look like `tox devenv -e units-devel devel-experiments-venv`. It might be useful to mention in the context of troubleshooting tests in a venv directly.